pom.xml

```diff
@@ -3,11 +3,11 @@
   <parent>
     <groupId>org.jenkins-ci.plugins</groupId>
     <artifactId>plugin</artifactId>
-    <version>3.5</version>
+    <version>3.50</version>
   </parent>
 
   <properties>
-    <jenkins.version>1.642.3</jenkins.version>
+    <jenkins.version>2.60.3</jenkins.version>

```

Comment:
Not sure this is appropriate, the plugin on 1.x Jenkins master won't be able to update anymore.
However, with current setting, it allow update on 1.x master and won't affect the update on 2.x master.
So, is there any reason we must set the min version of Jenkins to 2.60 except for Java 8 ?